base/stream.jl

```diff
@@ -1098,6 +1099,12 @@ for (x, writable, unix_fd, c_symbol) in
             ($f)($(writable ? :write : :read))
             return (read, write)
         end
+        function ($f)(::DevNull)
+            nulldev = @static Sys.iswindows() ? "NUL" : "/dev/null"
+            handle = open(nulldev, write=$writable)
+            $(_f)(handle, isnull=true)
```

Comment:
Just `$x = devnull` right after here instead of the extra keyword argument? Seems cleaner to me.

Reply:
agree - the argument was also to close `handle`, so I added that too.